pyproject.toml

```
@@ -0,0 +1,8 @@
+[tool.black]
+line-length = 99
+target-version = ['py36', 'py37', 'py38']
+extend-exclude = '''
+# A regex preceded with ^/ will apply only to files and directories
+# in the root of the project.
+/(_vendor|devenv|tests)/
```

I think the tests should be included in the move to use black.

```suggestion
/(_vendor|devenv)/
```

---

Makes perfect sense, I don't remember why I included `tests` here...

---

Done